composite_purchase_order.json

```diff
@@ -66,6 +70,34 @@
       "description": "indicates this purchase order should be re-encumbered each fiscal year. Only applies to ongoing orders",
       "type": "boolean"
     },
+    "renewal_cycle": {
+      "description": "length of the renewal cycle",
+      "type": "string",
+      "enum": [
+        "Two Years",
+        "One Year",
+        "3 Months",
+        "6 Months",
+        "1 Month",
+        "Manual"
+      ]
+    },
+    "renewal_interval": {
+      "description": "renewal interval (in days?)",
+      "type": "integer"
+    },    
+    "renewal_review_period": {
```

Comment:
Should this be `review_period` because that's how it was earlier? As long as it generates correct POJO, we should be good.

Reply:
yea , previously it was under renewal object so it was implied, now that it is moved to purchase order level, I added the prefix to make it more unambiguous